src/learn/qml/MuseScore/Learn/internal/ClassesPage.qml

```diff
@@ -61,13 +63,13 @@ FocusScope {
     }
 
     Rectangle {
-        anchors.top: parent.top
         anchors.left: parent.left
         anchors.leftMargin: root.sideMargin
         anchors.right: parent.right
         anchors.rightMargin: root.sideMargin
 
         height: authorInfo.height
+        width: Math.min(1000, parent.width - root.sideMargin)
```

Comment:
I'm afraid this conflicts with `anchors.right: parent.right`. Normally, you either specify the width, or two anchors. So in this case, you'll want to remove `anchors.right`.

Also, I think you should subtract `2 * root.sideMargin`, namely one for the left margin and one for the right margin.

Reply:
Should I remove `anchors.rightMargin` as well? I think the width logic is enough to set the right margin automatically.

Reply:
Yes, `anchors.rightMargin` without `anchors.right` has no effect.